catroid/src/main/res/values/strings.xml

```diff
@@ -41,6 +41,7 @@
     <string name="backpack">Backpack</string>
     <string name="packing">Pack</string>
     <string name="unpack">Unpack</string>
+    <string name="unpack_keep">Unpack and keep</string>
```

Comment:
Please remove `unpack_keep` and also [UiTestUtils.java#L1831](https://github.com/Catrobat/Catroid/blob/7f9dc2fcd5239be748a851a78210c385d22078af/catroid/src/androidTest/java/org/catrobat/catroid/uitest/util/UiTestUtils.java#L1831). It's not used anymore.